syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -206,7 +206,7 @@ def id(self):
         else:
             try:
                 return self._id
-            except:
+            except AttributeError:
```

Comment:
again should this be `if hasattr`?

Reply:
Same thing, I believe we get a good speedup with this but I might be wrong